qa/tasks/mgr/dashboard/test_cephfs.py

```diff
@@ -290,3 +290,20 @@ def test_listing_of_ui_api_ls_on_deeper_levels(self):
         ui_api_ls = self.ui_ls_dir('/pictures', 0)
         self.assertEqual(api_ls, ui_api_ls)
         self.rm_dir('/pictures')
+
+    def statfs(self, path):
+        params = {'path': path}
+        data = self._get("/api/cephfs/{}/statfs".format(self.get_fs_id()),
+                         params=params)
+        self.assertStatus(200)
+        self.assertIsInstance(data, dict)
+        return data
+
+    def test_statfs(self):
+        self.statfs('/')
+
+        self.mk_dirs('/animal')
+        date = self.statfs('/animal')
+        self.assertEqual(date['bytes'], 0)
+        self.assertEqual(date['files'], 0)
+        self.assertEqual(date['subdirs'], 1)
```

Comment:
Can you add some data and files as well to see they are reflected? This is essential to make sure we cover every part of the code.

Reply:
Okay, could you provide me call api that add data and file. I have not found it.

Reply:
Hmm, there aren't any api that can add data and/or file; so you'd need to add them in https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/services/cephfs.py.

For creating helpers you'd need to take help of https://github.com/ceph/ceph/blob/main/src/pybind/cephfs/cephfs.pyx. Use `self.cfs.open()` to create and open files(don't forget to close them) or use `self.cfs.mknod()` to create block or character special file. To add data to files use `self.cfs.write()`

Reply:
got it